Gemfile

```diff
@@ -58,3 +58,6 @@ gem 'net-pop'
 gem 'net-smtp'
 
 gem 'carrierwave'
+
+gem 'i18n_generators'
```

Comment:
自動生成に使うだけのものですし、developmentモードでのみ有効になるようにしましょう。

READMEにも言及されてますねー。
https://github.com/amatsuda/i18n_generators#installation

> Add 'i18n_generators' to your Gemfile, then bundle. Restricting the group to :development would be a good idea, since the generator would probably be used only in development mode.